contributing.md

```diff
@@ -0,0 +1,80 @@
+> # Meraki Frontend Repository
+
+This repository contains the source code for the Meraki Frontend. The Meraki Frontend is a web application that allows users to learn python , spoken english and other things.
+
+The Deployed Links For Meraki.
+
+[Meraki Production ]("https://merakilearn.org")
+
+[Meraki Development ]("https://bhanwari-devi.vercel.app")
+
+> ### Tech Stack
+
+- React
+- Redux
+- [MUI](https://mui.com/)
+- nodeJs (V14)
+
+> # Installation and Setup Guide
+
+1. Fork the repository
+2. create a Clone of the repository
+3. Make sure you have nodejs installed on your system with Version 14
+4. Run `npm install` to install all the dependencies .
+5. create a new .env file in root and add this code to it.
+
+   ```.env
+    REACT_APP_GOOGLE_CLIENT_ID = ''
+    REACT_APP_MERAKI_URL = 'https://api.merakilearn.org'
```

Comment:
Should this be `https://dev-api.merakilearn.org`?

Reply:
Yes